src/led_strip_rgb.py

```diff
@@ -0,0 +1,74 @@
+import neopixel, machine
+from time import sleep
+import time
+import uasyncio as asyncio
+from easing import *
+
+# Abstraction for light control - this gets used for the shade and base.
+class LedStripRGB:
+    def __init__(self, lamp, color, pin, num_pixels):
+        self.lamp = lamp
+        self.color = LedStripRGB.hex_to_rgb(color)
+        self.num_pixels = num_pixels
+        self.pin = pin
+        self.lock = asyncio.Lock()
+
+        self.pixels = neopixel.NeoPixel(machine.Pin(self.pin), self.num_pixels, bpp=3)
```

Comment:
isn't this the only change, and if so, could you just have an arg on the init that's bpp=4 which can be set to 3 instead? or is there other stuff in here

Reply:
Sending 4bpp values makes the strand malfunction. I think we can simply cherry pick this out for now